.gradle/8.10.2/dependencies-accessors/a8aa91efa41cfaba3fbe46dbe542780dff662146/sources/org/gradle/accessors/dm/LibrariesForLibs.java

```diff
@@ -0,0 +1,377 @@
+package org.gradle.accessors.dm;
```

Comment:
Ignore Gradle project-specific cache directory in .gitignore

Reply:
Updated .gitignore file in [https://github.com/my20yeezy/TicketBusManager/pull/1/commits/b77830f831adfd0d48b6fb89c2ea777bd9ad3196](url)

Reply:
still see this `.gradle` folder in git, it has to "disappear" on git side

Reply:
fixed in https://github.com/my20yeezy/TicketBusManager/pull/1/commits/d2c100e91b3babd60d3c47f389f799dcbd51c683